gitpod-io/gitpod
# [installer]: add customization function to ws-manager deployment

## Description

The deployment in `ws-manager` was missed out of #10857. This adds it in

## Related Issue(s)

Fixes #10853

## How to test

See [original instructions](https://github.com/gitpod-io/gitpod/pull/10857#issue-1282175614)

## Release Notes

```release-note
[installer]: add customization function to ws-manager deployment
```

## Documentation


## Werft options:
